packages/e2e-tests/config/setup-performance-test.js

```diff
@@ -32,13 +34,18 @@ beforeAll( async () => {
 
 	await trashAllPosts();
 	await trashAllPosts( 'wp_block' );
-	await setupBrowser();
+	await clearLocalStorage();
+	await setupPage();
 	await activatePlugin( 'gutenberg-test-plugin-disables-the-css-animations' );
-	await page.emulateMediaFeatures( [
-		{ name: 'prefers-reduced-motion', value: 'reduce' },
-	] );
 } );
 
 afterEach( async () => {
-	await setupBrowser();
+	// Clear localStorage between tests so that the next test starts clean.
+	await clearLocalStorage();
+	// Close the previous page entirely and create a new page, so that the next test
+	// isn't affected by page unload work.
+	await page.close();
+	page = await browser.newPage();
+	// Set up testing config on new page.
+	await setupPage();
```

Comment:
These last two commands feels like they should be in `beforeEach` no to avoid a lingering page at the end of all tests.

Reply:
Good point! I did that and removed the redundant `setupPage` call 👍